### ISSUE TITLE
Error if the url contains custom port

### ISSUE THREAD

**Issue by aledbf** [appc/docker2aci#12]
```
$ ./docker2aci internal-registry.com:5000/demo:v0.5
Downloading layer: 511136ea3c5a64f264b78b5433614aec563103b4d4702f3ba7d4d2698e22c158
Conversion error: error building layer: error generating the manifest: invalid char in ACName: :
```

https://github.com/appc/docker2aci/blob/master/lib/docker2aci.go#L421
https://github.com/appc/docker2aci/blob/master/lib/docker2aci.go#L472

**Comment by jonboulle** [comment]
This should be fixed by #13

**Comment by iaguis** [comment]
Actually, it should be fixed by #16 :relaxed:
